website/source/docs/installation.html.md

```diff
@@ -63,61 +63,31 @@ cluster, you should boot at least 3 nodes to keep etcd efficient
 (see [here](https://github.com/coreos/etcd/blob/v0.4.6/Documentation/optimal-cluster-size.md) for
 an explanation).
 
-### Dependencies
-
-Flynn uses Docker images to store job filesystems, and by default uses the AUFS filesystem driver. To
-check if your system supports AUFS, run this command:
-
-```
-$ sudo modprobe aufs
-```
-
-If you get no output, then AUFS is supported, but if you get `modprobe: FATAL: Module aufs not found.`
-then you need to run the following commands:
-
-```
-$ sudo apt-get update
-$ sudo apt-get install linux-image-extra-$(uname -r)
-```
-
-Flynn uses ZFS for persistent data volumes.  To install ZFS, run these commands:
-
-```
-apt-key adv --keyserver keyserver.ubuntu.com --recv E871F18B51E0147C77796AC81196BA81F6B0FC61
-echo deb http://ppa.launchpad.net/zfs-native/stable/ubuntu trusty main > /etc/apt/sources.list.d/zfs.list
-apt-get install -y ubuntu-zfs
-```
-
 ### Installation
 
-Flynn is available as a Debian package from the Flynn apt repository.
-
-First, add the Flynn repository key to your list of trusted apt keys:
+Download and run the Flynn installer script:
 
 ```
-$ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv BC79739C507A9B53BB1B0E7D820A5489998D827B
+$ sudo bash < <(curl -fsSL https://dl.flynn.io/install-flynn)
```

Comment:
@titanous ah yes I'm just used to always passing it to stdin rather than as an argument, updated.

WRT `curl | sudo bash`, when I am required to use `sudo` I like it being clear at the start of the command. Just a personal preference, happy to change if you think the alternative is clearer? Both side by side for reference:

```
sudo bash <(curl -fsSL https://dl.flynn.io/install-flynn)

curl -fsSL https://dl.flynn.io/install-flynn | sudo bash
```